packages/exiflooter/PKGBUILD

```diff
@@ -0,0 +1,53 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=exiflooter
+pkgver=33.a92e697
+pkgrel=1
+pkgdesc='Find geolocation on all image urls and directories also integrates with OpenStreetMap.'
+arch=('any')
+groups=('blackarch' 'blackarch-forensic')
+url='https://github.com/aydinnyunus/exifLooter'
+license=('Apache-2.0')
+depends=('perl-image-exiftool')
+makedepends=('git' 'go')
+source=("git+https://github.com/aydinnyunus/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  # use, if no version string provided neither in sources nor in git describe:
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+
+  # if tag exists, use this
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```

Comment:
remove the unneeded line, you already picked the git count method

Reply:
Removed